internal/services/containers/container_registry_token_password_resource.go

```diff
@@ -181,7 +182,10 @@ func (r ContainerRegistryTokenPasswordResource) Read() sdk.ResourceFunc {
 
 			tokenId := tokens.NewTokenID(id.SubscriptionId, id.ResourceGroup, id.RegistryName, id.TokenName)
 
-			pwds, err := r.readPassword(ctx, client, tokenId)
+			pwds, notFound, err := r.readPassword(ctx, client, tokenId)
+			if notFound {
```

Comment:
Can we combine this with the check on line 193
```suggestion
			if notFound || len(pwds) == 0 {
```